src/extensions/yfm/YfmTabs/YfmTabsSpecs/const.ts

```diff
@@ -1,8 +1,15 @@
 export enum TabsNode {
+    // tabs panel is common for tabs and radio tabs
+    TabPanel = 'yfm_tab_panel',
+
     Tab = 'yfm_tab',
     TabsList = 'yfm_tabs_list',
-    TabPanel = 'yfm_tab_panel',
     Tabs = 'yfm_tabs',
+
+    RadioTabs = 'yfm_r_tabs',
```

Comment:
`r` – can be interpreted as the `right` one, let's write the fullness – `yfm_radio_tabs`